### PR TITLE
Adds SVGs To Artist Header

### PR DESCRIPTION
**Pull request by ashleyjelks**
[Links to GROW-1431](https://artsyproduct.atlassian.net/browse/GROW-1431)
Kapture 2019-08-13 at 12 20 32

### CODE REVIEW

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx [hunk 1]
```diff
@@ -24,52 +33,90 @@ export class VanguardArtistWrapper extends React.Component<
...
   font-size: 100px;
   line-height: 1.35em;
+  color: ${color("white100")};
 `
 
+const ReadMoreButton = styled(Sans)<{ onClick: () => void }>`
+  text-transform: uppercase;
+`
 const ArtistContainer = styled(Box)`
-  /* override feature text drop-caps */
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx]: Why remove this comment?

**Comment by eessex** [comment]
Also we should add a test for the svg selection!

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx (hunk 1 above)
Comment (ashleyjelks) [reply src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx]: I just noticed that too, I think its a leftover when I was experimenting with bringing the container inside the component function.

**Comment by ashleyjelks** [comment]
> Nice! I would love to see some screenshots of this as well.

Screenshots instead of the video?

**Comment by ashleyjelks** [comment]
> Open to screenshots or video, but would like to see how this looks in some format

Hmm try refreshing your browser? I uploaded a video of the new work.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.24.4 :rocket:

**Pull request merged**